openshift/github-mirror.yaml

```diff
@@ -100,3 +106,9 @@ parameters:
   value: app-sre-bot:cs-sre-bot
 - name: GITHUB_MIRROR_URL
   value: https://github-mirror.stage.devshift.net
+- name: GITHUB_CACHE_TYPE
+  value: redis
+- name: REDIS_ENDPOINT
+  value: ghmirror-redis-noauth.huo5rn.ng.0001.use1.cache.amazonaws.com
```

Comment:
Ah, this should be an empty string here and actually defined on app-interface (so we don't expose our internal infra information)

Reply:
oops meant to ask you about that. will replace it with localhost rather than leave it empty

Reply:
it's technically a secret and i had it as a secretKeyRef, but moved it here since the aws instance wasn't defined through app-interface

Reply:
or maybe just temporarily edit the secret by hand and use here from the right place?

Reply:
sounds good.